OpenEdXMobile/src/main/java/org/edx/mobile/view/RegisterActivity.java

```diff
@@ -376,6 +376,13 @@ public void run() {
         }
     }
 
+    // make sure that on the login activity, all errors show up as a dialog as opposed to a flying snackbar
+    @Override
+    public boolean showErrorMessage(String header, String message) {
+        super.showErrorDialog(header, message);
```

Comment:
You don't need this since it seems that this method is not being called anywhere in the Activity now.